browser/src/map/handler/Map.Keyboard.js

```diff
@@ -594,18 +591,9 @@ L.Map.Keyboard = L.Handler.extend({
 		else if (ev.type === 'keydown') {
 			var key = ev.keyCode;
 			var map = this._map;
-			if (key in this._panKeys && !ev.shiftKey) {
-				if (map._panAnim && map._panAnim._inProgress) {
-					return;
-				}
-				map.fire('scrollby', {x: this._panKeys[key][0], y: this._panKeys[key][1]});
-			}
-			else if (key in this._panKeys && ev.shiftKey &&
-					!docLayer._textCSelections.empty()) {
-				// if there is a selection and the user wants to modify it
-				if (keyEventFn) {
-					keyEventFn('input', charCode, unoKeyCode);
-				}
+			if (['ArrowLeft', 'ArrowRight', 'ArrowUp', 'ArrowDown', 'Home', 'End'].includes(ev.code)) {
+				keyEventFn('input', charCode, unoKeyCode);
+				keyEventFn('up', charCode, unoKeyCode);
```

Comment:
On keydown we send `input` and `up`.
What does happen on `keyup` ?

Reply:
I think we only send it as "input", as is. I guess it's not keydown and it's not keyup. Also handled as input on the core side.

I must have forgotten this while testing. Thank you.